.ci/onnx/jenkins/Jenkinsfile

```diff
@@ -166,6 +174,13 @@ def main(String label, String projectName, String projectRoot, String dockerCont
                     RunToxTests(configurationMaps)
                 }
             }
+            catch(e) {
+                if ("$e".contains("143")) {
```

Comment:
Please add a comment explaning the reson for using "143" here.

Reply:
Done